.travis.yml

```diff
@@ -0,0 +1,180 @@
+dist: trusty
```

Comment:
Can you remove this file?  We have removed Travis CI integration in favor of Buildkite.